src/main.ts

```diff
@@ -45,3 +45,14 @@ import '@ionic/vue/css/display.css';
 
 /* Theme variables */
 import './theme/variables.css';
+
+
+/** Call `getRedirectResult` when app starts
```

Comment:
a Vue app is started in `src/main.ts`. And `getRedirectResult` is supposed to be called when the app starts